src/components/select-button/index.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import './index.css';
+
+interface SelectButtonProps {
+  label?: string;
+  data:string;
+  color?: 'black' | 'white';
+  backgroundColor?: 'blue' | 'green'; 
+  SetActive: () => void;
+  isActive?: boolean;
+}
+
+const SelectButton: React.FC<SelectButtonProps> = ({
+  label,
+  data,
+  color,
+  backgroundColor,
+  SetActive,
```

Comment:
SetActive > ilk büyük harf olmamalı setActive olur.
props color ve backgroundColor gerek yok,

sadece useState aktifleşmesi bırakacaksın <Button color ve backgroundColour için